Lesson-7/lesson-7-new/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <link rel="stylesheet" href="style.css">
+    <article class="top">
+    <div class="container_heading">
+        <img id="logo_img" class="logo" src="LinkedSprite.svg#logo_img" alt="logo">
+        <p class="stays"><a href=""> Stays</a> </p>
+        <p class="attractions"> <a href="">Attractions</a> </p>
+        <img class="night_img" src="LinkedSprite.svg#moon" alt="night">
+        <img class="account" src="LinkedSprite.svg#account" alt="accountcircle">
```

Comment:
Align the height
![image](https://user-images.githubusercontent.com/20189313/128646094-5f93cecd-89b6-49ce-baf2-e7591f485b53.png)